### DISCUSSION TITLE
Add openrouter

### DISCUSSION THREAD

**Discussion by seoeaa** [n4ze3m/dialoqbase#184] [Feature request]
Add openrouter there are many models, it even partially works but does not give any answers

**Comment by n4ze3m** [comment]
Hey, It's easy to add OpenRouter or any OpenAI-compatible API to Dialoqbase.

Just go to 'Settings' > 'Models' and click on 'Add Model.'

Here is an example of adding the OpenRouter model

https://github.com/n4ze3m/dialoqbase/assets/39720973/6153c8a0-8769-44ce-b64e-d2abb01acf42

**Reply by seoeaa** [reply]
thanks, I did the same thing, but it gave me an error, I’ll check it again using your video

**Reply by n4ze3m** [reply]
use the following URL as the base: `https://openrouter.ai/api/v1`, without any additional slashes

**Reply by seoeaa** [reply]
There was an error processing your request.

**Reply by seoeaa** [reply]
Please add OPENAI_API_KEY to your .env file

**Reply by seoeaa** [reply]
probably because of the Embedding model it doesn’t work

**Reply by seoeaa** [reply]
Yes, it worked, I had to choose another model probably because of the Embedding model
